lib/index.js

```diff
@@ -212,23 +212,29 @@ Latencies
  */
 const listHandler = async function listHandler(options) {
   const listedFuncs = await list();
+  const rawData = Object.keys(listedFuncs).map(key => ({
+    name: key,
+    lastDeployed: listedFuncs[key].tags.latest.modifiedAt,
+    expiration: listedFuncs[key].tags.latest.expiration,
+  }));
   if (options.json) {
-    const rawData = Object.keys(listedFuncs).map(key => ({
-      name: key,
-      lastDeployed: `[${listedFuncs[key].tags.latest.modifiedAt}]`,
-    }));
     logger.info(JSON.stringify(rawData));
-  } else {
-    const aggr = {};
-    Object.keys(listedFuncs).forEach((key) => {
-      aggr[key] = listedFuncs[key].tags.latest.modifiedAt;
-    });
-    if (Object.keys(aggr).length > 0) {
-      logger.info(columnify(aggr, {
-        columns: ['Function', 'Last Deployed'],
-        config: { Function: { minWidth: 25 } },
-      }));
-    }
+  } else if (rawData.length > 0) {
+    const columns = rawData.some(d => d.expiration)
```

Comment:
Please always keep the same columns -- it's less surprising.

Reply:
done